app/scripts/controllers/metametrics.js

```diff
@@ -315,6 +319,24 @@ export default class MetaMetricsController {
     this._identify(allValidTraits);
   }
 
+  // It sets an uninstall URL ("Sorry to see you go!" page),
+  // which is opened if a user uninstalls the extension.
+  updateExtensionUninstallUrl(participateInMetaMetrics, metaMetricsId) {
+    // TODO: Change it to the right URL once it's available.
+    const extensionUninstallUrl =
+      'https://swap.metaswap.codefi.network/tracking/uninstall';
```

Comment:
lets make this url a constant at the top of this file.

Reply:
i'm working with website team to define endpoint as `https://metamask.io/uninstalled`. 
Can track [status of ticket](https://github.com/ConsenSys/dx-team/issues/1339) for Website team, but not yet implemented (as of this comment)